metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -202,16 +229,30 @@ func CreateStackdriverSink(uri *url.URL) (core.DataSink, error) {
 		stackdriverClient: stackdriverClient,
 	}
 
+	// Register Prometheus metrics
```

Comment:
`Register sink metrics` or `Register sink performance metrics` explain it better